src/Algebra/Graph/Relation/Symmetric/Internal.hs

```diff
@@ -0,0 +1,193 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module     : Algebra.Graph.Relation.Symmetric.Internal
+-- Copyright  : (c) Andrey Mokhov 2016-2019
+-- License    : MIT (see the file LICENSE)
+-- Maintainer : andrey.mokhov@gmail.com
+-- Stability  : unstable
+--
+-- This module exposes the implementation of derived binary relation data types.
+-- The API is unstable and unsafe, and is exposed only for documentation. You
+-- should use the non-internal module "Algebra.Graph.Relation.Symmetric" instead.
+-----------------------------------------------------------------------------
+
+module Algebra.Graph.Relation.Symmetric.Internal (
+    -- * Implementation of derived binary relations
```

Comment:
Out-of-date comment?

Reply:
Yes. Updated it to `symmetric binary relations`